control/items_control.txt

```diff
@@ -54,7 +54,6 @@ all 0 1 0
 506 0 1 0 # Green Potion
 501 25 1 0 # Red Potion
 502 25 1 0 # Orange Potion
-504 25 1 0 # White Potion
```

Comment:
whit potion is missing

Reply:
oh dear for some random reason it was duplicated, and i don't know how i've deleted both of them

today at night i will change this